src/components/Auth/SignupOption.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+interface SignupOptionProps {
+  type: string;
+}
+function SignupOption(props: SignupOptionProps) {
+  const { type } = props;
+
+  const getSignOption = (type: string) => {
+    switch (type) {
+      case 'nickname':
+        return <StyledBtn>중복확인</StyledBtn>;
+      case 'emailConfirm':
+        return <StyledBtn>확인</StyledBtn>;
+      case 'password':
+        return <StyledNotice>‘영문 소문자 + 숫자’ 포함하여 8글자 이상 15자 미만</StyledNotice>;
+      default:
+        return null;
+    }
+  };
```

Comment:
```suggestion
  const getSignOption = (type: string) => {
    switch (type) {
      case 'nickname':
        return <StyledBtn tabIndex={-1}>중복확인</StyledBtn>;
      case 'emailConfirm':
        return <StyledBtn tabIndex={-1}>확인</StyledBtn>;
      case 'password':
        return <StyledNotice>‘영문 소문자 + 숫자’ 포함하여 8글자 이상 15자 미만</StyledNotice>;
      default:
        return null;
    }
  };
```
웹에서 여러개의 폼을 입력할 때 사용자들은 tab을 이용해 다음 입력 창으로 이동하는데 
이름에서 닉네임으로 tab을 사용해 넘어갈 때 중복확인에 걸리게 되어서 불편한 감이 있더라구요.

> MDN: 음의 정숫값(보통 tabindex="-1")은 연속 키보드 탐색으로 접근할 수는 없으나 JavaScript나 시각적(마우스 클릭)으로는 포커스 가능함을 뜻합니다. 보통 JavaScript를 사용한 위젯의 접근성 확보를 위해 사용합니다.

한번 직접 적용해보시고 차이를 확인해보면 좋을 것 같아요!